.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
+  "printWidth": 80,
+  "singleQuote": false,
+  "semi": true
+}
```

Comment:
Perhaps this change is something to discuss in a separate PR?

Reply:
Oh I see, this is to match the current settings, and override any user-level config that a developer might have...?

Reply:
Yeah my `~/.prettierrc` is set to the opposite of all of these, so running prettier on anything exploded the diffs. I ran `npm run prettify` after adding this and it changed nothing, except the `npm run check:format` on CI still failed. Not sure why...